memcache/memcache.go

```diff
@@ -171,6 +177,24 @@ type conn struct {
 	c    *Client
 }
 
+// Statistics is a record of memcached usage stats.
+type Statistics struct {
+	// Counter of cache hits.
```

Comment:
Write complete sentences.

See https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

// Hits is ...

(ending in a period)